### ISSUE TITLE
Beeping after firing

### ISSUE THREAD

**Issue by javier96s** [t238/Digital_Trigger_Unit#11]
Im getting a error:
I start using semi normally, until I hear a beep after each shot and doesn't let me shoot faster than that, auto still works nice 
Until it stops working and just beeps in long beeps
Anyone had this problem?

**Comment by patrickwieth** [comment]
yeah have this problem, troubleshoot manual does not explain this situation... no solution
